api/api/settings.py

```diff
@@ -151,7 +152,8 @@
 CORS_ORIGIN_WHITELIST = [
     'https://master.d2xlug9wi9lkht.amplifyapp.com',
     'http://localhost:3000',
-    'http://127.0.0.1:3000'
+    'http://127.0.0.1:3000',
+    'http://0.0.0.0:8000'
```

Comment:
Should this be here at all? CORS_ORIGIN means it's the URL of the client, but as this ends in 8000 it seems like the URL of your server.